pages/dkp/kommander/2.2/install/networked/index.md

```diff
@@ -94,6 +94,12 @@ helmrelease.helm.toolkit.fluxcd.io/traefik-forward-auth-mgmt condition met
 helmrelease.helm.toolkit.fluxcd.io/velero condition met
 ```
 
+If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress", you can trigger a reconciliation of the `HelmRelease` using the following command:
```

Comment:
```suggestion
If you find any `HelmReleases` in a "broken" release state such as "exhausted" or "another rollback/release in progress" (to check the `HelmReleases` with statuses, use `kubectl get helmreleases -A`), you can trigger a reconciliation of the `HelmRelease` using the following command:
```

Reply:
made a new section to check HR in https://github.com/mesosphere/dcos-docs-site/pull/4310/commits/edd9bc416d22f30b43cfc5fbda38493433cef88b